### DISCUSSION TITLE
strange error message

### DISCUSSION THREAD

**Discussion by FraxinusErit** [Pryaxis/TShock#2034] [Q&A]
I have been getting this error message, and I thought I would send it in case it is important. I am currently running the Tshock pre-release 11 on my raspberry pi 3 B+. I have occasional desync issues, and I was wondering if this error message has anything to do with it.

UpdateManager warning: Error: SecureChannelFailure (the authentication or decryption has failed.)
-> the authentication or decryption has failed.
-> the authentication or decryption has failed.
-> the authentication or decryption has failed.
retrying in 5 minutes.

and then it keeps sending the message every five minutes.

**Comment by moisterrific** [comment]
UpdateManager is what is supposedly check for updates to TShock and notify the server owner if a new version is available. Don’t think it has to do with the desync issues.

**Reply by moisterrific** [reply]
Can you post additional detail about your server setup? Is it SSC? How many players were on? Any anomalies seen in the logs other than this?
